modules/rostests/kmtests/ntos_ob/ObSecurity.c

```diff
@@ -8,6 +8,18 @@
 #include <kmt_test.h>
 #include "../ntos_se/se.h"
 
+// FIXME
+NTSYSAPI
+NTSTATUS
+NTAPI
+ZwQueryInformationProcess(
```

Comment:
You want a `#include <ndk/psfuncs.h>` here: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/kmtests/include/kmt_platform.h;hb=7b29f4d975a8bf5b196d9ca1903ca05829aa32bc#l27 ... I hope

Reply:
Doesn't work, already tried.
Hence the "FIXME" because my point wasn't trying to fix the usage of ZwQueryInformationProcess in kmtest.

Reply:
Here's a proper fix btw: https://pastebin.com/TpTpuHs6